### ISSUE TITLE
Compile the neural network and fit the data for the model training

### ISSUE THREAD

**Issue by alvinadar** [alvinadar/Dividend-Yield-Prediction-using-ANN#2]
Use -> Adam Optimizer 
mean squared error -> Loss Function
